test/rule-matches/not-html-matches.js

```diff
@@ -5,20 +5,21 @@ describe('not-html-matches', function() {
   var rule;
 
   beforeEach(function() {
-    rule = axe._audit.rules.find(function(rule) {
-      return rule.id === 'valid-lang';
-    });
+    axe.teardown();
```

Comment:
Why not put this in `afterEach`?

Reply:
I tried but it said that axe was already setup for the first test.

Reply:
PR for axe.teardown. If that gets merged first we can update this one https://github.com/dequelabs/axe-core/pull/2755